erpnext_documentation/www/docs/user/manual/en/accounts/articles/immutable-ledger-in-erpnext.md

```diff
@@ -0,0 +1,31 @@
+<!-- add-breadcrumbs -->
+# Immutable Ledger In ERPNext
+
+> Introduced in Version 13
+
+A major change has been introduced in ERPNext from version 13 onwards. This changes the way Accounting Ledger (General Ledger) and Stock Ledger works in ERPNext. There are multiple reasons why ledgers should be immutable. To list a few
+
+* Reposting future entries is computationally expensive.
```

Comment:
```suggestion
* Reposting future entries is computationally expensive, i.e., slows down other transactions.
```

Reply:
It doesn't slows down other transactions, the backdated transaction itself is computationally expensive as all future transactions needs to be updated